docs/topics/api/scanners.rst

```diff
@@ -0,0 +1,22 @@
+============
+Scanners
+============
```

Comment:
The change should be added to https://github.com/mozilla/addons-server/blob/master/docs/topics/api/overview.rst#v4-api-changelog too

Reply:
done